spec/models/form526_submission_spec.rb

```diff
@@ -700,7 +697,7 @@
 
         it 'calls confirmation email job with correct personalization' do
           allow(Form526ConfirmationEmailJob).to receive(:perform_async) do |*args|
-            expect(args[0]['first_name']).to eql('firstname')
+            expect(args[0]['first_name']).to eql('BEYONCE')
```

Comment:
This `expect` is not realistic since `first_name` is always `upcase`d by `get_first_name`.